tutorials/cloudsdk-vdm-generator/cloudsdk-vdm-generator.md

```diff
@@ -0,0 +1,369 @@
+
+2. Add the business object to the catalog
+
+    Assignment to a catalog can be done via the `Maintain Catalogs` option in the initial screen of the custom business object. We assign this object to the catalog `Employee – Master Data`.
+
+    ![Business Object Catalog](Business-Object-Catalog.png)
+
+    After publishing the object in the catalog, the newly created application is available in the Fiori Launchpad. In the next step, we will evaluate this application and create the data using the generated UI.
+
+3. Add data via UI in SAP S/4HANA Cloud
+
+    In the Fiori Launchpad, we can now see the newly created application:
+
+    ![Fiori Launchpad](Fiori-Launchpad.png)
+
+    We add the following few records. This data will be later accessed in our Java application:
+
+    ![Data](Data.png)
+
+    We can now easily access the created data via the UI in Fiori Launchpad. We now need to enable access to the corresponding API from our Java application that we will build. For that purpose, we need a custom communication scenario and a corresponding communication arrangement.
+
+4. Create custom communication scenario, user, system, and arrangement
+
+    We create the communication scenario, assigning the generated social network service as an inbound service, as it will be called from external systems, such as the local machine (for integration testing) and SAP Cloud Platform applications.
+
+    ![Communication Scenario](Communication-Scenario.png)
+
+    With the custom communication scenario in place, you can now create a communication arrangement based on the scenario. The creation of communication arrangement for custom OData API does not differ from the one for standard OData services.
+
+    The blog [Deep Dive 1 with SAP Cloud SDK: An End-to-End Story on the Virtual Data Model using the Business Partner API](https://blogs.sap.com/2017/09/15/step-12-with-sap-s4hana-cloud-sdk-consuming-business-api-from-s4hana-via-odata/) provides a good practical introduction on how to create the communication system, user, and arrangement in order to enable access to APIs from the local machine, which is sufficient for the example given in this blog post.
+
+    Also, please refer to this [blog post](https://blogs.sap.com/2017/09/15/what-is-what-your-quick-reference-to-communication-management-and-identity-access-management-artifacts-in-s4hana/) to get overview on the concept of SAP S/4HANA Cloud communication and integration artifacts.
+
+5. Evaluate the generated OData service
+
+    Now, when we have enabled the generated custom OData service, let us take a look at its metadata. The metadata of the OData service is represented by an `edmx` file that can be accessed by adding /$metadata to the OData service base path. We use the following URL:
```

Comment:
/$metadata should be in back ticks -- as well as code, files, folders, URLS. Please take a look through the tutorial for other cases.